plover/assets/main.json

```diff
@@ -97774,6 +97807,7 @@
 "SPHEU/TPHAR/-S": "seminars",
 "SPHEU/TROP/K-L": "semitropical",
 "SPHEUBG": "systemic",
+"SPHEUFD": "dismissed",
```

Comment:
I understand. I got inspired by this stroke however:
```
"STKAOEUD": "decide",
```
The word "decide" is probably used more often though. I'll remove this stroke.